### ISSUE TITLE
Upstream Trainable hugely reduce accuracy score for SID for both hubert and w2v2

### ISSUE THREAD

**Issue by david-wei-01001** [s3prl/s3prl#583]
Dear s3prl team,
I want to use the "upstream trainable"  to improve the hubert and w2v2 accuracy on voxceleb's SID task by finetuning the upstream model.

However, when I run the following command:
```bash
python3 run_downstream.py -m train -n fintuned_w2v2Exp -u wav2vec2 -d voxceleb1 --upstream_trainable

python3 run_downstream.py -m train -n fintuned_hubertExp -u hubert -d voxceleb1 --upstream_trainable
```
with hyperparameter: step = 200000. lr = 1.0e-1, 
the accuracy goes down dramatically comparing to the frozen version:
w2v2: 73.7% -> 0.25%
hubert: 81.6% -> 0.07%
